quickstart/101-traffic-manager-external-endpoint/main.tf

```diff
@@ -0,0 +1,55 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = random_pet.rg_name.id
+  location = var.resource_group_location
+}
+
+resource "random_string" "azurerm_traffic_manager_profile_name" {
```

Comment:
Why do you use random_string in some of these examples instead of random_pet?

Reply:
We've used random_string in every example where a name is needed. We've used random_pet only for the rg. I think it just kind of happened organically. You mentioned in one PR that the entity name should be realized via random_string and I've done it that way ever since.